### ISSUE TITLE
vim-pandoc breaks wikilinks in vimwiki

### ISSUE THREAD

**Issue by mickmcq** [vim-pandoc/vim-pandoc#329]
I have set

let g:vimwiki_list = [{'path': '~/vimwiki/', 'path_html': '~/vimwiki_html/', 'syntax': 'markdown', 'ext': '.md'}]

in my .vimrc so that my vimwiki files are markdown. Sometime between May and September 2019, wikilinks stopped working when I have the vim-pandoc plugin loaded. I believe that it worked before that.

**Comment by alerque** [comment]
I think you need to set the syntax as "pandoc", which is distinct from "markdown".

**Comment by mickmcq** [comment]
I did try setting the syntax as pandoc but vimwiki complained that that was an illegal value.

**Comment by fmoralesc** [comment]
Indeed, it seems like vimwiki uses its own markdown syntax system and the 'syntax' key for g:vimwiki_list simply enables that for the file. I am actually not sure that pandoc is loaded at all. Can you check that the value of `b:pandoc_loaded` is `1` for the vimwiki files?

**Comment by mickmcq** [comment]
b:pandoc_loaded does return 1 in the vimwiki index file.

**Comment by fmoralesc** [comment]
Have you tried disabling vim-pandoc's hypertext module?

**Comment by mickmcq** [comment]
I put

```vim
let g:pandoc#modules#disabled = ["hypertext","formatting","folding","command","menu","keyboard","bibliographies","completion","toc","spell"]
```

in my `.vimrc` after first just trying hypertext, but to no evident effect.
